src/tests/functional/shared_test_classes/src/single_layer/softmax.cpp

```diff
@@ -15,7 +15,7 @@ namespace subgraph {
 std::string SoftMaxLayerTest::getTestCaseName(const testing::TestParamInfo<SoftMaxTestParams>& obj) {
     ElementType netType, inType, outType;
     InputShape shapes;
-    size_t axis;
+    int64_t axis;
```

Comment:
Also create new test class for Softmax8 and keep existing one for Softmax1. Each plugin would have additional test case for Softmax8 instead of updating existing one.

Reply:
Now separate tests for Softmax1 and Softmax8 are supported.

The only clarification I need is which one to make default version called SoftMaxLayerTest -- either for opset1 or opset8.

In my view, the latest version of the operation should be tested by the SoftMaxLayerTest, and opset1 version by SoftMaxLayerTest_op1 or similar.

On the other hand, if we want this change without modifying existing plugins (that expect that SoftMaxLayerTest works with opset1), we should do vice versa.

Please advice!

Reply:
Please use the original name for Softmax-1 and  add SoftMax8LayerTest for new one.

Reply:
Done